### PR TITLE
feature(functions): add function to_nullable and assume_not_null

### PR DESCRIPTION
**Pull request by sundy-li**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

## Changelog

- New Feature

## Related Issues

Related to  #5922

### CODE REVIEW

File: docs/doc/30-reference/20-functions/120-other-functions/to_nullable.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+select to_nullable(3);
+
++----------------+
+| to_nullable(3) |
++----------------+
+|              3 |
++----------------+
+```
+
+```
```
Comment (b41sh) [inline docs/doc/30-reference/20-functions/120-other-functions/to_nullable.md:37]:
```sql

File: common/functions/src/scalars/others/assume_not_null.rs [hunk 2]
```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 Datafuse Labs.
```
Comment (b41sh) [inline common/functions/src/scalars/others/assume_not_null.rs:1]: Copyright 2022 Datafuse Labs.

File: common/functions/src/scalars/others/to_nullable.rs [hunk 3]
```diff
@@ -0,0 +1,75 @@
+// Copyright 2021 Datafuse Labs.
```
Comment (b41sh) [inline common/functions/src/scalars/others/to_nullable.rs:1]: Ditto

File: docs/doc/30-reference/20-functions/120-other-functions/to_nullable.md (hunk 1 above)
Comment (b41sh) [reply docs/doc/30-reference/20-functions/120-other-functions/to_nullable.md:37]: Missing `sql` character after quotes

**Review by BohuTANG — APPROVED** [review]

**Pull request merged**